Integrations/EWSv2/EWSv2.py

```diff
@@ -1851,6 +1877,30 @@ def mark_item_as_read(item_ids, operation='read', target_mailbox=None):
                                 marked_items)
 
 
+def get_item_as_eml(item_id, target_mailbox=None):
+    account = get_account(target_mailbox or ACCOUNT_EMAIL)
+    item = get_item_from_mailbox(account, item_id)
+
+    if not isinstance(item, Message):
+        return_error(item)
```

Comment:
what is the type of item here?
return_error should have a readable message

Reply:
Not necessary, removed it.